docs/content/development/extensions-core/druid-lookups.md

```diff
@@ -0,0 +1,122 @@
+# Cached Lookup Module
+
+## Description
+This module provides a one to one caching mechanism for a specific lookup table source like RDBMS or distributed file systems.
```

Comment:
suggest changing to 

`This module provides a per-lookup caching mechanism for JDBC data sources.` or simliar